app/(technician)/visit/[id]/layout.tsx

```diff
@@ -1,11 +1,10 @@
 import BackButton from "@/components/navigation/BackButton";
 import NavigationBar from "@/components/navigation/NavigationBar";
+import { ReactNode } from "react";
 
-export default function TechnicianVisit() {
+export default function Layout({children}: {children: ReactNode}) {
   return <>
     <NavigationBar title="Technician Visit" button={<BackButton />} />
-    <main className="px-6 pt-6 pb-16">
-
-    </main>
+    {children}
```

Comment:
Okay, this is a little confusing... How many different `main`'s are we going to have?

Is this intentional?